lib/msal-common/src/request/ScopeSet.ts

```diff
@@ -21,29 +21,49 @@ export class ScopeSet {
     // Boolean denoting whether scopes are required. Usually used for validation.
     private scopesRequired: boolean;
 
-    constructor(inputScopes: Array<string>, appClientId: string, scopesRequired: boolean) {
+    constructor(
+        inputScopes: Array<string>,
+        appClientId: string,
+        scopesRequired: boolean,
+        nonSPA?: boolean
+    ) {
         this.clientId = appClientId;
         this.scopesRequired = scopesRequired;
+
         // Filter empty string and null/undefined array items
-        const filteredInput = inputScopes ? StringUtils.removeEmptyStringsFromArray(inputScopes) : inputScopes;
+        const filteredInput = inputScopes
+            ? StringUtils.removeEmptyStringsFromArray(inputScopes)
+            : inputScopes;
+
         // Validate and filter scopes (validate function throws if validation fails)
         this.validateInputScopes(filteredInput);
-        const scopeArr = filteredInput ? StringUtils.trimAndConvertArrayEntriesToLowerCase([...filteredInput]) : [];
+
+        const scopeArr = filteredInput
+            ? StringUtils.trimAndConvertArrayEntriesToLowerCase([
+                ...filteredInput
+            ])
+            : [];
```

Comment:
This is arguably less readable when split up over multiple lines.

Reply:
auto correct by "prettify". We can change the linter rules as we see it before final merge.

Reply:
Edited